src/GraphQLEditor/GqlEditor.less

```diff
@@ -1,3 +1,5 @@
+@import '~@gen3/ui-component/dist/css/base.less';
```

Comment:
Stop using `base.less` file and use `--g3-color__xxx` instead of `@xxx`? (and I promise I will remove this `base.less` file later some day 😂